Specs/Core/loadJsonpSpec.js

```diff
@@ -0,0 +1,40 @@
+/*global defineSuite*/
+defineSuite([
+    'Core/loadJsonp',
+    'Core/RequestErrorEvent'
+], function(
+    loadJsonp,
+    RequestErrorEvent) {
+    "use strict";
+    /*global jasmine,describe,xdescribe,it,xit,expect,beforeEach,afterEach,beforeAll,afterAll,spyOn*/
+
+    it('throws with no url', function() {
+        expect(function() {
+            loadJsonp();
+        }).toThrowDeveloperError();
+    });
+
+    it('returns a promise that resolves when the request loads', function() {
+        var testUrl = 'http://example.invalid/testuri';
+        var promise = loadJsonp(testUrl).then(function(data){
+            expect(data).toEqual({
+                good : 'data'
+            });
+        }).otherwise(function(error) {
```

Comment:
Since you aren't using the `otherwise` block, you can get rid of it.  Also, return the promise at the end of this function.  All of our asynchronous unit tests need to return the promise they create.